src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.logo__img {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, apply styles using class selectors to avoid potential conflicts and improve the reusability of your CSS.